modules/mailer/mailer.go

```diff
@@ -32,8 +32,8 @@ func NewMessageFrom(to []string, from, subject, body string) *Message {
 	msg.SetHeader("To", to...)
 	msg.SetHeader("Subject", subject)
 	msg.SetDateHeader("Date", time.Now())
-	msg.SetBody("text/plain", body)
```

Comment:
Should we do replacement here? Because it is possible that email body can multiple lines from anywhere...